src/meta-srv/src/handler/collect_stats_handler.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::VecDeque;
+
+use api::v1::meta::HeartbeatRequest;
+use common_telemetry::debug;
+use dashmap::mapref::entry::Entry;
+use dashmap::DashMap;
+
+use super::node_stat::Stat;
+use crate::error::Result;
+use crate::handler::{HeartbeatAccumulator, HeartbeatHandler};
+use crate::metasrv::Context;
+
+pub struct CollectStatsHandler {
+    max_size_per_key: usize,
```

Comment:
"size" is meaningless here

```suggestion
    max_cached_stats_per_key: usize,
```